backend/src/main/java/com/ody/mate/domain/Mate.java

```diff
@@ -64,6 +64,11 @@ public Mate(Meeting meeting, Member member, Nickname nickname, Location origin,
         this(null, meeting, member, nickname, origin, estimatedMinutes);
     }
 
+    public boolean isAttended(Meeting otherMeeting) {
+        long myMeetingId = this.meeting.getId();
+        return myMeetingId == otherMeeting.getId();
+    }
```

Comment:
객체에게 물어보도록 한 점이 좋네요 ~

[제안]
Long 타입인 meeting_id를 long으로 언박싱한 이유가 있나요?

아래 처럼 한줄로 작성해도 가독성 좋으것 같은데 어떻게 생각하시나요 ?
```suggestion
    public boolean isAttended(Meeting otherMeeting) {
        return this.meeting.getId() == otherMeeting.getId();
    }
```

Reply:
좋아요! 더 간결해진 것 같네요